client/src/components/ProfilePage/index.js

```diff
@@ -108,25 +209,34 @@ const ProfilePage = (props) => {
           <Typography>{user.password}</Typography>
         </AccordionDetails>
       </Accordion>
+      <h1 className="Title">
+        <StyledText>Your listed items</StyledText>
+      </h1>
+
+      <Grid item xl={7} lg={9} md={9} sm={9}>
+            {loading ? (
+              <Loading />
+            ) : (
+              <ItemList items={listings} goToItem={goToItem} />
+            )}
+          </Grid>
 
       <Button
         className={classes.button}
         variant="contained"
         color="secondary"
-        onClick={console.log("Log out was pushed")}
-        onClick={() => props.history.push("/login")}
+        onClick={() => {user.username = ""
```

Comment:
if you grab setUser from Context, then you could simply do setUser({}) and it will accomplish all of this.  State variables are meant to be immutable, so doing it this way might lead to a bug down the line.

Reply:
Fixed it so now it uses setUser instead!